Source/ThirdParty/easy_profiler/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 2.8.12.1)
+project(easy_profiler CXX)
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+set(EASY_PROGRAM_VERSION_MAJOR 1)
+set(EASY_PROGRAM_VERSION_MINOR 3)
+set(EASY_PROGRAM_VERSION_PATCH 0)
+set(EASY_PRODUCT_VERSION_STRING "${EASY_PROGRAM_VERSION_MAJOR}.${EASY_PROGRAM_VERSION_MINOR}.${EASY_PROGRAM_VERSION_PATCH}")
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/bin)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/bin)
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/bin)
+
```

Comment:
These output overrides cause an error when building for multiple platforms as each platform's binaries will be in the same location, static easy_profiler libs for all platforms are written to:

```%ATOMIC_ROOT%/Source/ThirdParty/easy_profiler/easy_profiler_core/Release```

instead of for example:

```%ATOMIC_ROOT%/Artifacts/Build/Mac/Source/ThirdParty/easy_profiler/easy_profiler_core/Release```
```%ATOMIC_ROOT%/Artifacts/Build/IOS/Source/ThirdParty/easy_profiler/easy_profiler_core/Release```
```%ATOMIC_ROOT%/Artifacts/Build/Android/Source/ThirdParty/easy_profiler/easy_profiler_core/Release```

Disabling these output directory overrides, and ```./Build_AtomicEditor.sh --with-android --with-ios``` works.  I think this is a fine solution, though should put // ATOMIC BLOCKS and a description of why disabled.  This isn't standard CMake for a third party library and should probably be addressed upstream.